source/_integrations/group.markdown

```diff
@@ -205,6 +205,22 @@ services:
       type: string
 {% endconfiguration %}
 
+## Services
+
+This integration provides the following services to modify groups and a service to reload the configuration without restarting Home Assistant itself.
```

Comment:
Have you tried them?

As you will notice, it doesn't work on all groups...

Reply:
I haven't. I just did a quick test from the front end and it looks like it only supports the old style groups. Is that right?

If so, is it worth moving this to the **Old style groups** section or just binning the change?

Reply:
Its only old-style groups indeed.

Reply:
I have moved it into the old style group section. Hopefully it will help people using that method for creating groups and address the associated issue: https://github.com/home-assistant/home-assistant.io/issues/20414

Thanks @frenck